src/ubx.h

```diff
@@ -571,13 +569,17 @@ class GPSDriverUBX : public GPSHelper
 public:
 	GPSDriverUBX(Interface gpsInterface, GPSCallbackPtr callback, void *callback_user,
 		     struct vehicle_gps_position_s *gps_position,
-		     struct satellite_info_s *satellite_info);
+		     struct satellite_info_s *satellite_info,
+		     uint8_t dynamic_model);
```

Comment:
I actually did that initially, but assumed QGC was the only one setting it to 2?